<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Number of populations</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_met">
     <property name="text">
      <string>Number of metiers</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Number of Benthos func. grp.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Number of Size Groups</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Implicit Stocks level1 (absolute abundance unknown)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="m_implicit_pops"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Implicit Stocks level2 (but relative abundance used anyway)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="m_implicit_pops_level2">
     <property name="text">
      <string>not implemented</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Grouped TACs groups</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="m_grouped_tacs"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>NBCP coupled pop idx</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="m_nbcp_coupling_pops"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>calib the other landings per stock</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" rowspan="2">
    <widget class="QLineEdit" name="m_calib_oth_landings"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>calib weight-at-szgroup per stock</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="m_calib_weight_at_szgroup"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>calib the cpue multiplier per stock</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLineEdit" name="m_calib_cpue_multiplier"/>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Interesting Harbours</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLineEdit" name="m_int_harbours"/>
   </item>
   <item row="13" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="nbpops"/>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="nbmets"/>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="nbbenthospops"/>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="szGroups">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>14</number>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QToolButton" name="readFromTree">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="qtgui.qrc">
       <normaloff>:/icons/database.png</normaloff>:/icons/database.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qtgui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
